internal/install/recipe_installer_guided.go

```diff
@@ -107,7 +111,7 @@ func (i *RecipeInstaller) guidedInstall(ctx context.Context, m *types.DiscoveryM
 	if i.ShouldInstallIntegrations() {
 		log.Debugf("Installing integrations")
 		if err = i.installRecipes(ctx, m, selectedIntegrations); err != nil {
```

Comment:
should we just change this line to `_ = i.installRecipes(ctx, m, selectedIntegrations)`?

Reply:
That's not a bad idea.  I'm now seeing the debug log that comes after this, so, should we log something about receiving errors here?

Reply:
I've updated the logic here to keep the ErrInterrupt handled correctly.